site/content/kapp/_index.html

```diff
@@ -87,22 +86,9 @@ <h2>Basic Usage</h2>
 # ... or templated with ytt
 $ kapp -y deploy -a my-app -f <(ytt -f ./examples/simple-app-example/config-1.yml){{< / highlight >}}
     </div>
-    <div class="text-content" id="playground">
-        <h2>Playground and Example Workflow</h2>
-        <p>You can follow along in Katacoda Kubernetes playground while going through below workflow. To set up playground:</p>
-        <ol>
-            <li><a href="https://www.katacoda.com/courses/kubernetes/playground" target="_blank">Open Katacoda Playground</a></li>
-            <li>Click "Start scenario" button</li>
-            <li>Focus in "Terminal Host 1" prompt</li>
-            <li>Click "launch.sh". It will set up Kubernetes cluster</li>
-            <li>Click "kubectl cluster-info" to ensure that kubernetes cluster is set up successfully</li>
-            <li>Install kapp in playground before continuing with the workflow</li>
-        </ol>
-        {{< highlight bash-plain "" >}}
-$ wget -O- https://carvel.dev/kapp-install-katacoda.sh | bash
-$ cd carvel-kapp/
-$ kapp version{{< / highlight >}}
-        <p>And now onto using <code>kapp</code>:</p>
+    <div class="text-content" id="example">
```

Comment:
nit: Not sure if we need this id `example`.

Reply:
Removed